SPECS/ModemManager/ModemManager.spec

```diff
@@ -66,8 +66,8 @@ make  %{?_smp_mflags} check
 %{_datadir}/locale/*
 %{_datadir}/bash-completion/*
 %{_datadir}/gir-1.0/ModemManager-1.0.gir
+%{_datadir}/ModemManager/*
 %exclude %{_datadir}/icons
-%exclude %{_datadir}/ModemManager/mm-dell-dw5821e-carrier-mapping.conf
```

Comment:
Did you check on why this was excluded in the first place, and why we might want to include it now?

Reply:
The file is no longer generated and throws a FNF error when excluded.